x/simulation/simulate.go

```diff
@@ -74,6 +75,15 @@ func SimulateFromSeed(
 		return true, fmt.Errorf("must have greater than zero genesis accounts")
 	}
 
+	// remove module account address if they exist in accs
```

Comment:
The call to `initChain` returns a new set of accounts which will happen to include module accounts. This seems to be because it calls [AppStateFromGenesisFileFn](https://github.com/cosmos/cosmos-sdk/blob/280734d0e37f16b725906ec2b151c0b20e7dfaf1/simapp/utils.go#L67). This behavior has been tested and results in the sim bug